### PR TITLE
5 our story redesign

### LINKED ISSUE #5: The "our story" page should contain the printing process information

### PR DESCRIPTION
**Pull request by luantran**
# Issue
Closes #5 

#Changes in this pull request
Added "Our story " page template and sections template

#Screenshot or Gif of the pull request
screenshot from 2018-02-13 17-36-49

### CODE REVIEW

File: theme/config/settings_schema.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
     "theme_name": "Brooklyn",
     "theme_author": "Shopify",
     "theme_version": "4.2.1",
-    "theme_documentation_url": "https://help.shopify.com/manual/using-themes/themes-by-shopify/brooklyn",
-    "theme_support_url": "https://support.shopify.com/"
+    "theme_documentation_url": "https:\/\/help.shopify.com\/manual\/using-themes\/themes-by-shopify\/brooklyn",
```
Comment (belljustin) [inline theme/config/settings_schema.json:7]: Why are all the forward slashes in all these URLs escaped now?

File: theme/sections/our-story-blog.liquid [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+    {% if section.settings.title != blank %}
+    <div class="section-header text-center">
+      <h2 class="h1 section-header__title">{{ section.settings.title | escape }}</h2>
+      <hr class="hr--small">
+    </div>
+    {% endif %}
+    {% assign count = 0%}
+
+    {% for article in blog.articles reversed%}
```
Comment (belljustin) [inline theme/sections/our-story-blog.liquid:17]: It's really too bad Shopify doesn't seem to have a feature for adding more fields to the "page" menu but this is a creative solution to the problem of adding content to the page!

File: theme/config/settings_data.json [hunk 3]
```diff
@@ -1 +1,647 @@
...
+          "title": "Image with text",
+          "text": "<p>Pair large text with an image to give focus to your chosen product, collection, or blog post. Add details on availability, style, or even provide a review.<\/p>",
+          "button_label": "",
+          "button_link": ""
+        }
+      },
+      "our-story-blog": {
+        "type": "our-story-blog",
+        "settings": {
+          "title": "Printing Process - BlogStyle",
```
Comment (belljustin) [inline theme/config/settings_data.json:284]: I'm concerned about having this info in a JSON file. Either we'll have to educate Tessa on how this works (i.e. installing theme-kit and uploading changes) or leave documentation for future devs that will take over the project

**Review by assetdesigns — APPROVED** [review]

File: theme/config/settings_data.json (hunk 3 above)
Comment (luantran) [reply theme/config/settings_data.json:284]: I did not modify this file manually, Shopify does it for me. 

This JSON file is updated everytime one updates the content through the shopify interface:
screenshot from 2018-02-14 19-57-28

So any data you change in that left panel, then in updates it in the setting_data.json

File: theme/sections/our-story-blog.liquid (hunk 2 above)
Comment (luantran) [reply theme/sections/our-story-blog.liquid:17]: I have talked to Tessa about the 2 methods I came up with, and we settled on using the sections way of doing it. Its the "proper" way to do it... I'll write a more detailed explanation of how the templating is gonna work

File: theme/config/settings_schema.json (hunk 1 above)
Comment (luantran) [reply theme/config/settings_schema.json:7]: I have no idea... Havent touched that file, Shopify must have changed it somehow

**Pull request merged**